core/commands/check_relevance_command.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require_relative '../models/network_settings'
+require_relative 'base_command'
+require_relative '../services/machine_configurator'
+
+# This class checks relevance network config
+class CheckRelevanceCommand < BaseCommand
+  AVAILABLE_TIME = 5
+  def self.synopsis
+    'Check for relevance of network_config file.'
+  end
+
+  def show_help
+    info = <<-HELP
+Check relevance command tries to connect to all nodes in the _network_config file.
+_network_config file is relevant, if the connection was successful to all nodes.
+    HELP
+    @ui.info(info)
+  end
+
+  def execute
+    if @env.show_help
+      show_help
+      return SUCCESS_RESULT
+    end
+    check_relevance
+  end
+
+  def check_relevance
+    return Result.error("#{@args} does not exist") unless File.file?(@args)
+
+    network_settings = NetworkSettings.from_file(@args)
+    all_nodes = network_settings.node_name_list
+    machine = MachineConfigurator.new(@ui)
+    all_nodes.each do |node|
```

Comment:
You must check only nodes that are specified in the _network_configuration file. It is possible that user wants only a part of nodes available.

In other words: we do not need to check that all nodes from the configuration are present in _network_config file. But we do need to check that all nodes that are present in _network_config file are reachable via data that we provide to them.